R/comp_corr.R

```diff
@@ -1,11 +1,12 @@
 #' Computing corr
 #'
-#' This function computes the correlation upon picking the best lag and
-#' adjusting the data for lags
+#' This function computes the weighted correlation with penalty (no penalty if not lagged) after the lags
+#' applied to the dataset and timepoints using prep.data() and best.lag() function.
 #'
-#' @param data a matrix with rows representing genes and columns representing
+#'
+#' @param data a lagged matrix with rows representing genes and columns representing
 #' different timepoints (NA's added when lags are needed)
-#' @param time a vector which contains all the timepoints represented in the data
+#' @param time a lagged matrix which contains all the timepoints represented in the data, NA is introduced when it is lagged
```

Comment:
Should we describe the dimensions of this matrix like we do for `data`?

Reply:
I will fix this

Reply:
We still want to change this before merging.

Reply:
Kudos @thevaachandereng, @agitter; the explanation in `comp.corr()` is aligned with how I expected it to behave. 

Thanks!